core/src/main/scala/org/ensime/config/richconfig.scala

```diff
@@ -32,9 +34,18 @@ package object richconfig {
         name.startsWith("scala-library") && name.endsWith(".jar")
       }
 
+    def findProject(f: FileObject)(implicit vfs: EnsimeVFS) = {
```

Comment:
ok, now **that** is the sort of thing that should go into a comment. Anything "unexpected" should be documented, otherwise the reader just thinks you missed something :wink: